VARIABLES.md

```diff
@@ -838,15 +878,15 @@ Default:  "{{ monitoring_interceptors_enabled }}"
 
 ### ksql_user
 
-Only use to customize Linux User ksqlDB Service runs with. User must exist on host.
+Only use to customize Linux User ksqlDB Service runs with. Default user is cp-ksql
```

Comment:
Modify this variable to customize the Linux User that the ksqlDB Service runs with. Default user is cp-ksql.